logistics/scripts/modules/path-resolver.Tests.ps1

```diff
@@ -0,0 +1,55 @@
+BeforeAll { 
+    Import-Module "$PSScriptRoot/utility/cross-platform.psm1"
+    Import-Module -Force -Scope Global ($PSCommandPath.Replace('.tests.ps1', '.psm1'))
+}
+Describe 'Select-RepositoryResolvedFiles' {
+    It 'Returns a valid and reachable path' {
+        $RepositoryFiles = Select-RepositoryResolvedFiles "logistics"
+        Test-Path $RepositoryFiles.FullName | Should -Be $true
+    }
+}
+Describe 'Select-CumulativeRepositoryResolvedItems' {
+    It 'Returns a valid and reachable path' {
+        $RepositoryFiles = Select-CumulativeRepositoryResolvedItems "logistics"
+        Test-Path $RepositoryFiles.FullName | Should -Be $true
+    }
+}
+
+Describe 'Get-RootPath' {
+    It 'Returns a valid and reachable Root path' {
+        $getRootPath = Get-RootPath
+        Test-Path $getRootPath.Path | Should -Be $true
+    }
+}
+Describe 'Get-CorePath' {
+    It 'Returns an ODS Core path' {
+        $getCorePath = Get-CorePath
+        Test-Path $getCorePath.Path | Should -Be $true
+        $getCorePath.Path.TrimEnd([IO.Path]::DirectorySeparatorChar) | Should -BeLike "*Ed-Fi-ODS"
+    }
+}
+Describe 'Get-RepositoryResolvedPath' {
+    It 'Return paths to specified files, checking all repositories' {
+        Get-RepositoryResolvedPath
+    }
+}
+Describe 'Get-RepositoryRoot' {
+    It 'Returns valid and reachable repository roots' {
+        $getRepositoryRoot = Get-RepositoryRoot
+        Test-Path $getRepositoryRoot[0] | Should -Be $true
+        Test-Path $getRepositoryRoot[1] | Should -Be $true
+    }
+}
+Describe 'Get-RepositoryNames' {
+    It 'Returns the repositoryNames variable' {
+        Get-RepositoryNames | Should -Not -BeNullOrEmpty
+    }
+}
+
+
+
+
+
+AfterAll {
+    Remove-Item "$($pwd.Path)/Application/EdFi.Ods.Extensions.Tests" -Recurse -Force
```

Comment:
Since this directory is no longer created in the BeforeAll section, this Remove-Item has to be deleted as well, otherwise the test fails in Linux

Reply:
Didn't catch that, thanks